dotbot/dispatcher.py

```diff
@@ -18,7 +18,10 @@ def __init__(
     ):
         self._log = Messenger()
         self._setup_context(base_directory, options)
-        plugins = plugins or []
+        if plugins == None:
+            plugins = Plugin.__subclasses__()
```

Comment:
This will circumvent the configuration option `--disable-built-in-plugins`. It will re-enable plugins that the user may have requested be disabled.